styles.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+    font-family: "Figtree-ExtraBold";
+    src: url('./assets/fonts/static/Figtree-ExtraBold.ttf') format('truetype');
+}
+
+@font-face {
+    font-family: "Figtree-Medium";
+    src: url('./assets/fonts/static/Figtree-Medium.ttf') format('truetype');
+}
+
+body {
+    background-color: rgb(244, 208, 78);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-height: 100vh;
+}
+
+img {
+    border-radius: 10px;
+}
+
+h1, h2, p {
+    line-height: 150%;
+    letter-spacing: 0px;
+}
+
+h1 {
+    font-size: 24px;
+    font-family: "Figtree-ExtraBold", sans-serif;
+}
+
+h2 {
```

Comment:
In my comments above I recommended changing `<h2>` to `<p>`, which will break this rule. So need to tweak the CSS accordingly

In larger designs, there's usually mismatch between the semantic meaning of an element and the visual design. For example, something which is an `<h2>` may be styled to be visually smaller than an `<h3>` somewhere else on the page, in spite of the page's actual semantic hierarchy. This mismatch can make element selectors tricky to work with